modules/book.js

```diff
@@ -0,0 +1,63 @@
+const bookTitle = document.querySelector('.book-title');
+const authorName = document.querySelector('.author-name');
+const article = document.querySelector('.book-details');
+export class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.displayBookDetails = this.displayBookDetails.bind(this);
+    this.bookArray = [];
+    if (localStorage.getItem('bookArray') != null) {
+      this.bookArray = JSON.parse(localStorage.getItem('bookArray'));
+    } else {
+      this.bookArray = [{
+        title: 'Mistborn',
+        author: 'Brandon Sanderson',
+      }];
+    }
+  }
+
+  // update method
+  updateLocalStorage() {
+    localStorage.setItem('bookArray', JSON.stringify(this.bookArray));
+  }
+
+  // remove method
+  removeBook(bookToRemove, obj) {
+    article.removeChild(bookToRemove);
+    this.bookArray.splice(this.bookArray.indexOf(obj), 1);
+    this.updateLocalStorage();
+  }
+
+  // display method
+  displayBookDetails(obj) {
+    const bookContainer = document.createElement('div');
+    bookContainer.classList.add('book');
+    if (this.bookArray.indexOf(obj) % 2 === 1) {
+      bookContainer.classList.add('white');
+    }
+    const bookId = `a${Math.floor(Math.random() * 100000)}`;
+    bookContainer.id = bookId;
+    const buttonId = `a${Math.floor(Math.random() * 100000)}`;
+    bookContainer.innerHTML = `
+            <p class='book_title'>'${obj.title}' by ${obj.author} </p>
+            <button class='remove_button'
+            id='${buttonId}'>Remove</button>
+            `;
+    article.append(bookContainer);
+    const removeBtn = document.querySelector(`#${buttonId}`);
+    removeBtn.addEventListener('click', (evt) => {
+      evt.stopPropagation();
+      this.removeBook(bookContainer, obj);
+    });
+  }
+
+  addBook() {
+    const newBook = new Book(bookTitle.value, authorName.value);
+    this.bookArray.push(newBook);
+    this.displayBookDetails(newBook);
+    bookTitle.value = '';
+    authorName.value = '';
+    this.updateLocalStorage();
+  }
```

Comment:
- [ ] Great job on making the necessary updates to your code as the previous reviewer requested! 👍  However, there is one more adjustment that needs to be made to align with the project requirements. Instead of using the traditional function syntax, it is recommended to use arrow syntax for functions. Arrow functions provide a concise and more modern way of defining functions in JavaScript. They have a simplified syntax and lexical scoping of `this`, which can be beneficial in certain contexts. 🙏 

Please update your code to use arrow syntax for your functions according to the project requirements. This will ensure consistency and adherence to the specified coding style. Keep up the excellent work! 👏🌟

Reply:
Thanks for your remarks, Resolved as you suggested.